src/main/java/org/sqlite/jdbc3/JDBC3ResultSet.java

```diff
@@ -474,32 +450,7 @@ public Time getTime(int col) throws SQLException {
      * @see java.sql.ResultSet#getTime(int, java.util.Calendar)
      */
     public Time getTime(int col, Calendar cal) throws SQLException {
-        checkCalendar(cal);
```

Comment:
(Duplicate) It seems like this moves the driver away from the JDBC spec by ignoring the user supplied Calendar/Timezone information.